.github/workflows/test.yaml

```diff
@@ -5,32 +5,30 @@ jobs:
   lint:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
-      - name: Install Python
-        uses: actions/setup-python@v2
-        with:
-          python-version: 3.6
+      - uses: actions/checkout@v4
+      - name: Install uv
+        uses: astral-sh/setup-uv@v4
+      - name: Set up Python
+        run: uv python install
       - name: Install deps
-        run: pip install black flake8
+        run: uv sync --dev
       - name: Lint
-        run: flake8 src/ test/
+        run: uv run flake8 src/ test/
```

Comment:
If you're going for all the new hotness, might as well switch to `ruff` too :)

Reply:
good idea actually

Reply:
You will want to review their docs and turn on a lot of the rules sets via your pyproject.toml

Reply:
Going to look at upgrading the Django min version through here, will take that opportunity to turn on ruff rules that "make sense" along the way